### ISSUE TITLE
dll files needed

### ISSUE THREAD

**Issue by bruhmidium** [malad1211/Mania-Cat#8]
When i try to open it, it says i need 3 dll files to open it. I dont know what these dll files are,

**Comment by ghost** [comment]
Yeah It Says Need Msvcp140.dll in order to work. After downlaoding it from external site,the app wouldnt open and says an error occured.
![image](https://user-images.githubusercontent.com/82861574/121765344-d4f79f00-cb67-11eb-9d33-775cd434d664.png)

**Comment by janjb1212** [comment]
Yeah, Where are the files?

**Comment by faitalen** [comment]
This error occurs because you are missing these dll. files. You might want to download them and put them in your SysWOW64 folder if your operating system is 64 bit. I cant really find the dll file downloads but you might want to look on yt on how to return these files. Hope this helps.

**Comment by frosteelX** [comment]
you have to extract it from the .rar files

**Comment by xXGlitchedXx** [comment]
still doesnt work
